MultiplatformDemoWithSync/shared/src/commonMain/kotlin/io/realm/kotlin/demo/model/CounterRepository.kt

```diff
@@ -41,7 +42,7 @@ class CounterRepository {
     private var realm: Realm
     private val counterObj: Counter
 
-    private val app: App = App.create(AppConfiguration.Builder(MONGODB_REALM_APP_ID).build())
+    private val app: App = App.create(AppConfiguration.Builder(MONGODB_REALM_APP_ID).log(LogLevel.ALL).build())
```

Comment:
We should probably not add this log level here? It is a firehose for no real use?